docs/plugins/github-apps.md

```diff
@@ -46,9 +46,9 @@ You have to create the GitHub Application manually using these
 as GitHub Enterprise does not support creation of apps from manifests.
 
 Once the application is created you have to generate a private key for the
-application it in a `yaml` file.
+application and place it in in a YAML file.
```

Comment:
Think this needs fixing too @adamdmharvey

Reply:
thx! 😁